@navikt/core/react/src/util/i18n/locales/nn.ts

```diff
@@ -97,4 +97,24 @@ export default {
       reset: "Tilbakestill tidsperspektiv",
     },
   },
+  DatePicker: {
+    close: "Lukk",
```

Comment:
Is this generic enough to be under "global" ?

Reply:
Nvm see the todo under `nb.json` now. I think it makes sense to just add it to global 👍

Reply:
Yes, probably. I will review all the TODOs after all translations has been added.